su-template/default/layout/html/layout.erb

```diff
@@ -2,23 +2,28 @@
 <html>
 <head>
   <%= erb(:headers) %>
-  <link rel="shortcut icon" type="image/vnd.microsoft.icon"
-    href="https://help.sketchup.com/sites/all/themes/sketch_help/favicon.ico">
-  <link rel='stylesheet' type='text/css'
-    href='https://fonts.googleapis.com/css?family=Open+Sans:400,700,400italic'>
-
-  <script>
-    // Every time this page is loaded, it sends this action to SketchUp, telling
-    // SketchUp that a new page has loaded that has example snippets that should
-    // be replaced with code editors. Nothing happens if the page is loaded in a
-    // regular browser outside of the SketchUp client.
-    $( document ).ready(function() {
-      if (typeof sketchup == 'object') {
-         sketchup.page_loaded();
-      }
-    });
-  </script>
-
+  <!-- STYLESHEETS -->
+    <link rel="stylesheet" type="text/css"
+    href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,400italic" />
+<% stylesheets.each do |stylesheet| %>
+    <link rel="stylesheet" href="<%= url_for(stylesheet) %>" type="text/css" charset="utf-8" />
+<% end %>
+  <!-- SCRIPTS -->
+<%= erb :script_setup %>
+<% javascripts.each do |javascript| %>
+    <script type="text/javascript" charset="utf-8" src="<%= url_for(javascript) %>"></script>
```

Comment:
Yes, it is. This loop (and the one to write the stylesheet elements) was moved from `"headers.erb"` so as to provide a better ordering of elements in the `<head>` of all pages. **So once again, they are from the core template.**